examples/mnist/quantized.cpp

```diff
@@ -30,10 +33,11 @@
 using namespace tiny_dnn;
 using namespace tiny_dnn::activation;
 
-void construct_net(network<sequential>& nn) {
-    // connection table [Y.Lecun, 1998 Table.1]
+void construct_net(network<sequential> &nn) {
+// connection table [Y.Lecun, 1998 Table.1]
 #define O true
 #define X false
+    // clang-format off
```

Comment:
> I don't like tons of badly formatted code and I don't think anyone would correct it manually.
It would be great if clang-format can correct the style much better.